app/scrapers/bing.py

```diff
@@ -13,8 +13,6 @@ def __init__(self):
         self.defaultStart = 1
         self.startKey = 'first'
         self.name = 'bing'
-        self.videoKey = 'FORM'
-        self.imageKey = 'FORM'
```

Comment:
Both require at same time.

Reply:
No both of these keys are redundant in the search params, since anyway `imageURL` and `videoURL` are different. https://www.bing.com/images/search?q=hello and https://www.bing.com/images/search?q=hello&FORM=HDRSC2 give the same results and so do https://www.bing.com/videos/search?q=hello&FORM=HDRSC3 and https://www.bing.com/videos/search?q=hello. You can check my deployed URL if you don't believe me, it works perfectly 😄